src/external_api.py

```diff
@@ -0,0 +1,42 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+currency_api_key = os.getenv("CURRENCY_API_KEY")
+share_api_key = os.getenv("SHARE_API_KEY")
+currency_headers = {"apikey": f"{currency_api_key}"}
```

Comment:
Он и так приходит из `.env` строкой, так что f-строка линяя
```suggestion
currency_headers = {"apikey": currency_api_key}
```